chapter_appendix_math/distributions.md

```diff
@@ -1,14 +1,14 @@
 # Distributions
 :label:`sec_distributions`
 
-Now that we have learned about how to work with probability theory in both discrete and continuous setting, lets get to know some of the common random distributions encountered.  Depending on the area of machine learning we are working in, we may potentially need to be familiar with vastly more of these, or for some areas of deep learning potentially none at all.  This is, however, a good basic list to be familiar with.
+Now that we have learned about how to work with probability theory in both discrete and continuous setting, lets get to know some of the common random distributions encountered.  Depending on the area of machine learning we are working in, we may potentially need to be familiar with vastly more of these, or for some areas of deep learning potentially none at all.  This is, however, a good basic list to be familiar with.  Let's first import some common libraries.
 
 ```{.python .input}
 %matplotlib inline
 import d2l
 from IPython import display
-from mxnet import np, npx
-npx.set_np()
```

Comment:
Can we still use mxnet.np instead of numpy?

Reply:
Sadly I need to switch it back to base numpy as soon as I start sampling from distributions.  At the moment I think it is cleaner to do it here.